tests/sources/xml/test_dspace_dim.py

```diff
@@ -216,3 +216,48 @@ def test_dspace_dim_transform_with_optional_fields_missing_transforms_correctly(
         format="electronic resource",
         content_type=["Not specified"],
     )
+
+
+def test_get_contents_success(dspace_dim_record_all_fields):
+    assert DspaceDim.get_contents(dspace_dim_record_all_fields) == ["Chapter 1"]
+
+
+def test_get_contents_transforms_correctly_if_fields_blank(
+    dspace_dim_record_optional_fields_blank,
+):
+    assert DspaceDim.get_contents(dspace_dim_record_optional_fields_blank) == []
+
+
+def test_get_contents_transforms_correctly_if_fields_missing(
+    dspace_dim_record_optional_fields_missing,
+):
+    assert DspaceDim.get_contents(dspace_dim_record_optional_fields_missing) == []
+
+
+def test_get_dates_success(dspace_dim_record_all_fields):
+    assert DspaceDim.get_dates(dspace_dim_record_all_fields, "abc123") == [
```

Comment:
For field method `_success` tests, we should extract the values from the `_all_fields_transforms_correctly` test